pkg/messaging/grpc_proxy.go

```diff
@@ -72,7 +72,15 @@ func NewProxy(opts ProxyOpts) Proxy {
 
 // Handler returns a Stream Handler for handling requests that arrive for services that are not recognized by the server.
 func (p *proxy) Handler() grpc.StreamHandler {
-	return grpcProxy.TransparentHandler(p.intercept, p.resiliency, p.IsLocal,
+	return grpcProxy.TransparentHandler(p.intercept,
+		func(appID, methodName string) *resiliency.PolicyDefinition {
+			_, isLocal, err := p.isLocal(appID)
+			if err == nil && !isLocal {
```

Comment:
Won't this stop any policies from being executed on a callee app when Dapr invokes the app?

Reply:
My understanding is that policies are to be applied on the caller side only. I changed this because otherwise I wasn't able to get resiliency policies to be applied. But let me know if I'm wrong